virttest/virt_admin.py

```diff
@@ -868,5 +868,5 @@ def client_disconnect(server_name, client_id, **dargs):
     :param dargs: standardized virt-admin function API keywords.
     :return: CmdResult object
     """
-    cmd = "client_disconnect %s %s" % (server_name, client_id)
+    cmd = "client-disconnect %s %s" % (server_name, client_id)
```

Comment:
Thanks very much for your reviewing.
I am not very sure about what you mean. 
What I did was just fix a typo.